test/metabase/query_processor_test/generative_test.clj

```diff
@@ -0,0 +1,37 @@
+(ns metabase.query-processor-test.generative-test
+  "Query processor generative tests.
+
+  WIPWIPWIP"
+  (:require
+   [clojure.test :refer [deftest is testing]]
+   [environ.core :refer [env]]
+   [metabase.lib.convert :as lib.convert]
+   [metabase.lib.core :as lib]
+   [metabase.lib.metadata :as lib.metadata]
+   [metabase.lib.metadata.jvm :as lib.metadata.jvm]
+   [metabase.lib.test-util.generators :as lib.tu.gen]
+   [metabase.lib.test-util.random :as ra]
+   [metabase.query-processor :as qp]
+   [metabase.test :as mt]))
+
+(set! *warn-on-reflection* true)
+
+(deftest ^:parallel basic-gen-query-test
+  (when (not-empty (env :mb-test-run-generative-test))
```

Comment:
Won't this run even if you set `MB_TEST_RUN_GENERATIVE_TEST="false"` or `=0`? I'm not sure exactly what `environ` will do.